thunder/torch/__init__.py

```diff
@@ -3041,6 +3041,26 @@ def batch_norm(
 #
 
 
+@torchsymbol(torch.baddbmm, is_method=True)
+def baddbmm(
+    a: TensorLike, b1: TensorLike, b2: TensorLike, *, beta: float = 1.0, alpha: float = 1.0, out: TensorLike = None
+) -> TensorLike:
+    utils.check(out is None, lambda: "Non-None out is not supported", NotImplementedError)
+
+    utils.check_same_dtype(a, b1, b2)
```

Comment:
We will probably do shortcuts below, so we should probably check that all things are on the same device.